src/style.css

```diff
@@ -1,3 +1,72 @@
+html {
+  --color-col: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Arial, sans-serif;
+}
+
+.header {
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 20px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+}
+
+.nav__list {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  color: var(--color-col);
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+  text-decoration: none;
+  position: relative;
+}
+
+.nav__list li {
+  margin-right: 20px;
+}
+
+.nav__list li:last-child {
+  margin-right: 50px;
+}
+
+.nav:hover {
+  color: var(--color-col);
+}
+
+.apple {
+  color: var(--color-col);
+  border-bottom: 2px solid var(--color-col);
+  padding-bottom: 20px;
+}
+
+.header__logo {
+  max-width: 100%;
+  height: auto;
+  margin-left: 50px;
+}
+
+.nav__link {
```

Comment:
add correct color for text, and change it on hover as in design. read requirements